src/main/java/org/gridsuite/sensitivityanalysis/server/service/SensitivityAnalysisInputBuilderService.java

```diff
@@ -161,16 +166,17 @@ private Stream<IdentifiableAttributes> getIdentifiablesFromContainer(Sensitivity
         return listIdentAttributes.stream();
     }
 
-    private Stream<IdentifiableAttributes> getMonitoredIdentifiablesFromContainer(SensitivityAnalysisRunContext context, Network network, EquipmentsContainer filter, List<IdentifiableType> equipmentsTypesAllowed, Reporter reporter) {
-
-        List<IdentifiableAttributes> listIdentAttributes = goGetIdentifiables(filter, context.getNetworkUuid(), context.getVariantId(), reporter);
+    private Stream<IdentifiableAttributes> getMonitoredIdentifiablesFromContainer(SensitivityAnalysisRunContext context, Network network, List<EquipmentsContainer> filters, List<IdentifiableType> equipmentsTypesAllowed, Reporter reporter) {
+        List<String> containerNamesList = filters.stream().map(EquipmentsContainer::getContainerName).toList();
+        String containerNames = Arrays.toString(containerNamesList.toArray());
```

Comment:
You can refactor this part to avoid code duplication since it's commonly used with **getIdentifiablesFromContainer**
Proposition: 

```
 private String getContainerNames(List<EquipmentsContainer> filters) {
        List<String> containerNamesList = filters.stream().map(EquipmentsContainer::getContainerName).toList();
        return Arrays.toString(containerNamesList.toArray());
    }
```
Then use it here: 
`.withValue("name", getContainerNames(filters))`